### ISSUE TITLE
Automate the project workflow further

### ISSUE THREAD

**Issue by Idrinth** [Idrinth/api-bench#229]
**Is your feature request related to a problem? Please describe.**
A github workflow file should be created to automate the project status/field changes where sensible, since it can not be done via the default integrations properly.

**Describe the solution you'd like**
- on create issue:
  - add it to the project
  - add it to the backlog
- on assign issue
  - move it to in progress
- on create pull request draft
  - move it and it's linked issue to in progress
  - assign it's linked issue to the PR creator
- on create pull request
  - mark it as draft
  - move it to in progress
  - move it's linked issue to in progress
  - assign it's linked issue to the PR creator
- on pipeline in PR success && all checkboxes ticked
  - move the PR to in Review
  - request review from @Idrinth 
- on PR approved
  - move the PR to in Test
- on PR comment "test positive"
  - move the PR to to merge
- On PR merge
  - move the pr to done
  - move the issue to done

**Comment by RISHIKESHk07** [comment]
@Idrinth could i contribute on this issue

**Comment by Idrinth** [comment]
@RISHIKESHk07 yes, you are welcome. It's not high priority, but would be pretty useful :) Feel free to drop by our slack as well if you have questions, that aren't yet answered by the CONTRIBUTING.md - or just want to chat!

**Comment by RISHIKESHk07** [comment]
@Idrinth ,is there way a way to test the code for workflows  ?

**Comment by Idrinth** [comment]
> @Idrinth ,is there way a way to test the code for workflows ?

not really besides creating an empty project to try them out :(

**Comment by RISHIKESHk07** [comment]
Did add secret using ``` act -s GITHUB_TOKEN=[my-token] ``` , but still getting this error any idea why 
![image](https://github.com/Idrinth/api-bench/assets/117551967/069dbbaa-6018-442b-a2b8-78b6029f5668)

**Comment by Idrinth** [comment]
Would be willing to test it out on the repo here - should be unable to do any damage - feel free to open a PR with it.

**Comment by RISHIKESHk07** [comment]
@Idrinth go it to work(act) will try to test code locally for today if everything works out will put up a PR ,or else will test on the repo directly

**Comment by RISHIKESHk07** [comment]
on assign issue
move it to in progress , here in progress is a label ?

**Comment by Idrinth** [comment]
> on assign issue move it to in progress , here in progress is a label ?

it's a status field on the project - turns up in column 2 here: https://github.com/users/Idrinth/projects/2

**Comment by RISHIKESHk07** [comment]
@Idrinth are there examples of github actions  for working with projects , i am not able get this part to work

**Comment by RISHIKESHk07** [comment]
@Idrinth found a way without using gh client will try testing it , and pushing a PR , any suggestion from your side

**Comment by Idrinth** [comment]
> @Idrinth found a way without using gh client will try testing it , and pushing a PR , any suggestion from your side

great, thank you!
